lib/compiler/passes/generate-bytecode.js

```diff
@@ -362,13 +372,17 @@ function generateBytecode(ast) {
         env,
         action: node
       });
-      const functionIndex = addFunctionConst(Object.keys(env), node.code);
+      const match = node.match | 0;
+      const functionIndex = emitCall && match >= 0
```

Comment:
what does any of this mean?

Reply:
Imagine thinking that code where you have to look at bytecode output to understand what it does is acceptable, but calling other people's code low quality and sloppy.

This approach is unreadable and should not be accepted.